monai/application/datasets.py

```diff
@@ -16,6 +16,7 @@
 from typing import Any, Callable
```

Comment:
change application -> apps as we discussed

Reply:
updated in the latest commit.
Thanks.